tests/tests_unit/rules/test_models/test_dms_architect_rules.py

```diff
@@ -1059,7 +1059,12 @@ def invalid_container_definitions_test_cases() -> Iterable[ParameterSet]:
                     },
                 ]
             },
-            "views": {"data": [{"view": "WindTurbine", "class_": "WindTurbine"}]},
+            "views": {
+                "data": [
+                    {"view": "WindTurbine", "class_": "WindTurbine"},
+                    {"view": "sp_core:Asset", "class_": "sp_core:Asset"},
+                ]
+            },
```

Comment:
Now that all validation is done together all errors raise simultaneously, so needed to add `{"view": "sp_core:Asset", "class_": "sp_core:Asset"}`to avoid getting NonExistingViewError.